contracts/test/integration/utils/AllMainnetRoles.sol

```diff
@@ -35,12 +35,8 @@ contract IntegrationTestAllMainnetRoles is RoleTesting, AllRolesConfig {
         /// pcv guards
         allAddresses[4].push(MainnetAddresses.EOA_1);
         allAddresses[4].push(MainnetAddresses.EOA_2);
-        allAddresses[4].push(MainnetAddresses.EOA_3);
         allAddresses[4].push(MainnetAddresses.EOA_4);
 
-        /// pcv guard admin
-        allAddresses[5].push(MainnetAddresses.PCV_GUARD_ADMIN);
```

Comment:
this shouldn't be removed. pcv guard admin role is still given to pcv guard admin